src/main/proto/cosmas.proto

```diff
@@ -164,7 +164,8 @@ message ForcedFileCommitResponse {
 
 message RestoreDeletedFileRequest {
     ProjectInfo info = 1;
-    string fileId = 2;
+    string fileId = 2;    // for compatibility I can't rename it

```

Comment:
I thought it wass number that matters, not the name. 
https://stackoverflow.com/questions/45431685/protocol-buffer-does-changing-field-name-break-the-message

Reply:
ok, I ll test it